<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configurações</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="bb_okCancel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>481</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_vlcPath">
      <property name="text">
       <string>Caminho do executável do VLC win32:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_vlcPath">
      <property name="text">
       <string>&quot;C:\Program Files\VideoLAN\VLC\vlc.exe&quot;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>501</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_channel">
      <property name="text">
       <string>Canal de música</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_selfChannel">
      <property name="text">
       <string>Usar canal atual</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_channelList">
      <item>
       <property name="text">
        <string>Default Channel</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>501</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="label_language">
      <property name="text">
       <string>Idioma</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_language">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>pt-BR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>en-US</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_rcHost">
      <property name="text">
       <string>VLC rc host:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_rcHost">
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_rcPort">
      <property name="text">
       <string>porta:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_rcPort">
      <property name="text">
       <string>32323</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="pb_connectVlc">
      <property name="text">
       <string>Conectar ao VLC</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QPushButton" name="pb_enableBot">
      <property name="text">
       <string>Ativar bot</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7"/>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8"/>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bb_okCancel</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bb_okCancel</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_connectVlc</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>connectVlc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_enableBot</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>toggleBot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_selfChannel</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>useSelfChannel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_language</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>onLanguageChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>toggleBot()</slot>
  <slot>connectVlc()</slot>
  <slot>useSelfChannel()</slot>
  <slot>onLanguageChange(int)</slot>
 </slots>
</ui>
